protocol/communities/manager.go

```diff
@@ -47,27 +47,30 @@ var pieceLength = 100 * 1024
 
 const maxArchiveSizeInBytes = 30000000
 
+var memberPermissionsCheckInterval = 1 * time.Hour
```

Comment:
Wonder if we should go with a different interval here

Reply:
Thank you for sharing your thoughts on this Sam.
So the check is already happening when the membership list is updated (on handle request to join).

However, because the state of any member's wallet can change at any given time really, we have to regularly check on-chain if the all the required funds are still in each member's wallet.

In other words, with this we'd query various chains every hour.

Reply:
Ah, that is a very fair point. I was assuming permissions were soul bound but they may not be.

Reply:
Humans don't have souls.